docs/en/build-store/refresh-control.md

```diff
@@ -0,0 +1,240 @@
+---
+title: Validations and refresh control
+table_of_contents: true
+---
+
+# Validations and refresh control
+
+## What is refresh control?
+
+_“🎵 baby don't update me, don't update me no more 🎵”_
+
+
+Refresh control is a mechanism for controlling the revisions of a snap that will be offered as updates  to a device. A *gating snap* can control the revisions of a *gated snap* so that, if the *gating snap* is installed on a device, only revisions of the *gated snap* which have been validated by the publisher of the *gating snap* can be installed or refreshed to.
+
+A gating snap can control revisions of one or more gated snaps. Similarly, a gated snap can require verification from more than one gating snap, and a particular revision of the gated snap will only be installable or refreshable to, if it satisfies the constraints of all its gating snaps. However, because it can be confusing to device operators, use of multiple gating snaps to control one specific gated snap is not recommended. The best practice is to use a single gating snap for each gated snap.
+
+This is especially useful for device manufacturers to mediate releases of the *core* snap, allowing time for testing and validation of new revisions on their device models and compatibility with their software.  Note this is separate from the ability for device managers to control the schedule of refreshes and revision control via a snap proxy.
+
+
+## Preliminary setup
```

Comment:
If we use the term "refresh control scheme" as suggested above, this header could be made more specific: Setting the gating snap's _refresh control scheme_

Reply:
See previous comment about new terminology.